### DISCUSSION TITLE
Lookback Period

### DISCUSSION THREAD

**Discussion by krinya** [kevinzg/facebook-scraper#485] [Q&A] [answered]
Hi all,

I am not sure but I might seen it somewhere that I can specify a date that prefers to the lookback period of post on a page.

Is my memory right? I am not able to find it in the documentation.

Is there a way to do this?

Kind regards,
Kristóf

**Comment by neon-ninja** [comment, marked as answer]
Not a date per se, but you can pass a start_url. See https://github.com/kevinzg/facebook-scraper/issues/310#issuecomment-852652846
